packages/fxa-auth-server/lib/routes/subscriptions/paypal.ts

```diff
@@ -235,7 +235,7 @@ export class PayPalHandler extends StripeWebhookHandler {
         await this.paypalHelper.processInvoice({
           customer,
           invoice: latestInvoice,
-          ipaddress: request.info.remoteAddress,
+          ipaddress: request.app.clientAddress,
```

Comment:
This is an unrelated fix I'm sneaking in here -- we should be sending PayPal the client IP address, not the remote address. See `IPADDRESS` in the [`DoReferenceTransaction` docs](https://developer.paypal.com/api/nvp-soap/do-reference-transaction-nvp/).